source/content/plugins-known-issues.md

```diff
@@ -22,6 +22,16 @@ We do not prevent you from installing and using these plugins or themes. However
 
 If your work is already updated but still listed here, let us know so we can remove it, or [submit a pull request](https://github.com/pantheon-systems/documentation/edit/main/source/content/modules-plugins-known-issues.md).
 
+### Pantheon MU Plugin Compatibility Layer
```

Comment:
I'm wondering if the documentation for this feature belongs in a new h3 subsection here on this page instead: https://docs.pantheon.io/guides/wordpress-configurations/plugins#pantheon-must-use-plugin 

Then the known issues page could add an info alert that crosslinks?

Reply:
Since the compatibility layer was specifically built to handle the plugins on this page, I think it belongs here, but I think a note (and links back and forth) on the mu-plugin doc is probably a good idea 👍